src/components/gmServiceGmail.js

```diff
@@ -557,46 +557,49 @@ gmServiceGmail.prototype = {
           }
           
           try {
-            //var tbMatches = data.match(/\["tb",(?:.|\s)+?](?:\s]){2,}(?!\s,,)/g);   see: https://github.com/gmail-manager-community/gmail-manager-community/issues/4
-            var tbMatches = [];
+            var loc1 = data.indexOf("var VIEW_DATA=[[");
+            var loc2 = data.lastIndexOf("var GM_TIMING_END_CHUNK2");
+            var viewData = data.substring(loc1+14, loc2-2);
```

Comment:
@arix98 can you explain why +14 and -2?

and is there a way to get full data like fetched by addon? a URL or so?
I thought it was https://mail.google.com/mail/u/0/feed/atom, but seems not to be...